src/sgame/Beacon.cpp

```diff
@@ -474,10 +474,13 @@ namespace Beacon //this should eventually become a class
 			BG_BoundingBox( static_cast<class_t>( parent->client->ps.stats[ STAT_CLASS ] ), &mins, &maxs, nullptr, nullptr, nullptr );
 			BG_MoveOriginToBBOXCenter( center, mins, maxs );
 
-			// Also update weapon for humans.
 			if( parent->client->pers.team == TEAM_HUMANS )
 			{
+				// Also update weapon for humans.
 				ent->s.bc_data = BG_GetPlayerWeapon( &parent->client->ps );
+				// Nudge the center a bit upwards for human players, so we don't look at their butt too much
+				float height = maxs.z - mins.z;
+				center.z += 0.3f * height;
```

Comment:
Note also that the sgame change seems to have no effect in cgame. This makes me unsure of its usefulness. I include it here for consistency.

Reply:
I think .z is fine

Reply:
I think the sgame part of this commit should be removed. As stated in the OP, it is unnecessary. This is a change about how the beacon is graphically displayed; for the beacon entity it makes more since to just keep tracking the origin.

Reply:
Ok, removed the sgame part